core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/gceModels.scala

```diff
@@ -18,3 +19,29 @@ object GceInstanceStatus extends Enum[GceInstanceStatus] {
   case object Suspended extends GceInstanceStatus
   case object Terminated extends GceInstanceStatus
 }
+
+sealed abstract class GpuType extends Product with Serializable {
+  def asString: String
+}
+object GpuType {
```

Comment:
Is this from https://cloud.google.com/compute/docs/gpus?

`accelerator-optimized` machine type sounds interesting, maybe could be a future enhancement

Reply:
yea....so the `accelerator-optimized` are machine types, which we can consider adding in terra-ui in the future...I'm thinking we won't offer those in the beginning